cli/tests/pcluster/cli/test_entrypoint/TestParallelClusterCli/test_helper/pcluster-help.txt

```diff
@@ -1,50 +1,75 @@
 usage: pcluster [-h]
-                {create,update,delete,start,stop,status,list,instances,ssh,configure,export-cluster-logs,list-cluster-logs,get-cluster-log-events,build-image,delete-image,describe-image,list-images,export-image-logs,list-image-logs,get-image-log-events,version,list-clusters,dcv}
+                {list-clusters,create-cluster,delete-cluster,describe-cluster,update-cluster,describe-compute-fleet-status,update-compute-fleet-status,delete-cluster-instances,describe-cluster-instances,list-images,build-image,delete-image,describe-image,describe-official-images,status,configure,create,dcv-connect,delete,export-cluster-logs,get-cluster-log-events,instances,list-cluster-logs,list,ssh,start,stop,update,version,export-image-logs,get-image-log-events,list-image-logs}
                 ...
 
 pcluster is the AWS ParallelCluster CLI and permits launching and management
 of HPC clusters in the AWS cloud.
 
-positional arguments:
-  {create,update,delete,start,stop,status,list,instances,ssh,configure,export-cluster-logs,list-cluster-logs,get-cluster-log-events,build-image,delete-image,describe-image,list-images,export-image-logs,list-image-logs,get-image-log-events,version,list-clusters,dcv}
-    create              Creates a new cluster.
-    update              Updates a running cluster using the values in the
-                        config file.
-    delete              Deletes a cluster.
-    start               Starts the compute fleet for a cluster that has been
-                        stopped.
-    stop                Stops the compute fleet, leaving the head node
-                        running.
+optional arguments:
+  -h, --help            show this help message and exit
+
+COMMANDS:
+  {list-clusters,create-cluster,delete-cluster,describe-cluster,update-cluster,describe-compute-fleet-status,update-compute-fleet-status,delete-cluster-instances,describe-cluster-instances,list-images,build-image,delete-image,describe-image,describe-official-images,status,configure,create,dcv-connect,delete,export-cluster-logs,get-cluster-log-events,instances,list-cluster-logs,list,ssh,start,stop,update,version,export-image-logs,get-image-log-events,list-image-logs}
+    list-clusters       Retrieve the list of existing clusters managed by the
+                        API. Deleted clusters are not listed by default.
```

Comment:
Not sure if that should be a part of this pass (getting new CLI functionality working)? Seems separate since this PR is generally pretty generic relative to that particular data.

Reply:
ok let's track this as part of a separate activity/pr